rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -769,8 +771,6 @@ public MavenRepository normalizeRepository(MavenRepository originalRepository, M
                                             repository.getSnapshots(),
                                             repository.getUsername(),
                                             repository.getPassword());
-                                } else if (!e.isClientSideException()) {
-                                    return originalRepository;
```

Comment:
removing this feels a little cryptic, but I'm pretty sure it's no longer relevant?
eg, when it was written, the `e.isServerReached` call a few lines earlier did not exist:
https://github.com/openrewrite/rewrite/commit/3bcafba698a442818b740140ce42571a2a54dd7d#diff-a39c7415cc78264a78509aaaba13d76b4eeea28cdc5723e15615b7a8add5b325

Reply:
that said, tests pass -- ready for review @timtebeek